controllers/controller/workspacerouting/workspacerouting_controller.go

```diff
@@ -293,7 +302,10 @@ func (r *WorkspaceRoutingReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	if config.ControllerCfg.IsOpenShift() {
 		bld.Owns(&routeV1.Route{})
 	}
-	bld.WithEventFilter(routingPredicates)
+	if r.GetSolverFunc == nil {
```

Comment:
What is the reason to set up the default get solver func? It may lead to set up two controllers that listen to "default"/"embedded" routing classes.
What is instead we require it and explicitly specify it on our main.go, see like in the following https://github.com/sleshchenko/devworkspace-operator/commit/0a86eac28422cd3dd19cf997e5717cfeaaf21d73

Reply:
Ah I somehow missed that I could return an error here -- that makes much more sense.